politeiawww/cmd/politeiawww_dbutil/README.md

```diff
@@ -3,45 +3,111 @@
+connection at a time.
 
 
 ## Usage
 
 You can specify the following options:
 
-```
-    --testnet
-    Whether to interact with the testnet or mainnet database
+    Database options
+      -leveldb
+            Use LevelDB
+      -cockroachdb
+            Use CockroachDB
 
-    --datadir <dir>
-    Specify a different directory where the database is stored
+    Application options
+      -testnet
+            Use testnet database
+      -datadir string
+            politeiawww data directory
+            (default osDataDir/politeiawww/data)
+      -host string
+            CockroachDB ip:port 
+            (default localhost:26257)
+      -rootcert string
+            File containing the CockroachDB SSL root cert
+            (default ~/.cockroachdb/certs/clients/politeiawww/ca.crt)
+      -clientcert string
+            File containing the CockroachDB SSL client cert
+            (default ~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt)
+      -clientkey string
+            File containing the CockroachDB SSL client cert key
+            (default ~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key)
+      -encryptionkey string
+            File containing the CockroachDB encryption key
+            (default osDataDir/politeiawww/dbkey.json)
 
-    --dump [email]
-    Print the contents of the entire database to the console, or the
-    contents of the user, if provided.
+    Commands
+      -addcredits
+            Add proposal credits to a user's account
+            Required DB flag : -leveldb or -cockroachdb
+            LevelDB args     : <email> <quantity>
+            CockroachDB args : <username> <quantity>
+      -setadmin
+            Set the admin flag for a user
+            Required DB flag : -leveldb or -cockroachdb
+            LevelDB args     : <email> <true/false>
+            CockroachDB args : <username> <true/false>
+      -stubusers
+            Create user stubs for the public keys in a politeia repo
+            Required DB flag : -leveldb or -cockroachdb
+            LevelDB args     : <importDir>
+            CockroachDB args : <importDir>
+      -dump
+            Dump the entire database or the contents of a specific user
+            Required DB flag : -leveldb
+            LevelDB args     : <email>
+      -createkey
+            Create a new encryption key that can be used to encrypt data at rest
+            Required DB flag : None
+            Args             : <destination (optional)>
+                               (default osDataDir/politeiawww/dbkey.json)
+      -migrate
+            Migrate a LevelDB user database to CockroachDB
+            Required DB flag : None
+            Args             : None
 
-    --setadmin <email> <true/false>
-    Sets or removes the given user as admin.
+Examples:
 
-    --addcredits <email> <quantity>
-    Adds proposal credits to the given user.
+    $ politeiawww_dbutil -cockroachdb -setadmin username true
```

Comment:
Beginners might have trouble when running the commands on local env. I think a good approach is to give an example of how to execute commands using the `dbutil` interface. 

Like this:

```
If you are running commands locally, you must use the -testnet flag

$ politeiawww_dbutil -cockroachdb -setadmin -testnet username true
```

Reply:
I added a testnet example to the readme.